### PR TITLE
Bryan

### PR DESCRIPTION
**Pull request by bryancasler**

### CODE REVIEW

**Review by tjstalcup — APPROVED** [review]

File: src/index.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import { Options, App } from "@4site/engrid-common"; // Uses ENGrid via NPM
-// import { Options, App } from "../../engrid-scripts/packages/common"; // Uses ENGrid via Visual Studio Workspace
+// import { Options, App } from "@4site/engrid-common"; // Uses ENGrid via NPM
+import { Options, App } from "../../engrid-scripts/packages/common"; // Uses ENGrid via Visual Studio Workspace
```
Comment (fernanDOTdo) [inline src/index.ts:2]: We should undo this.

File: src/sass/main.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /* ENGRID */
-@import "@4site/engrid-styles"; // Uses ENGrid via NPM
-// @import "../../../engrid-scripts/packages/styles"; // Uses ENGrid via Visual Studio Workspace
+// @import "@4site/engrid-styles"; // Uses ENGrid via NPM
+@import "../../../engrid-scripts/packages/styles"; // Uses ENGrid via Visual Studio Workspace
```
Comment (fernanDOTdo) [inline src/sass/main.scss:3]: We should undo this.

File: src/templates/page-donation-aiusa.html [hunk 3]
```diff
@@ -145,6 +145,30 @@
       }
     </script>
 
+    <script>
```
Comment (fernanDOTdo) [inline src/templates/page-donation-aiusa.html:148]: Is this part a Test/WIP?

File: src/templates/page-donation-aiusa.html [hunk 4]
```diff
@@ -1491,13 +1521,64 @@ <h2 class="section-header">Payment Info</h2>
...
+                  style="min-height: 38px"
                 />
               </div>
             </div>
           </div>
           <div
             class="en__component en__component en__component--column en__component--advcolumn body-footer"
-          ></div>
+          >
+          <div class="en__component en__component--contactblock" style="">
```
Comment (fernanDOTdo) [inline src/templates/page-donation-aiusa.html:1534]: Should we create a separated file for this? It does not look like this is related to donations.

**Review by fernanDOTdo — CHANGES_REQUESTED** [review]

**Pull request merged**